plugin/src/App/StatusPages/Settings/init.lua

```diff
@@ -104,6 +104,14 @@ function SettingsPage:render()
 					layoutOrder = layoutIncrement(),
 				}),
 
+				SyncLock = e(Setting, {
+					id = "syncLock",
+					name = "Sync Lock",
+					description = "Toggle sync lock",
```

Comment:
Add `tag = "caution"` and add that tag type to the `TAG_TYPES` [here.](https://github.com/rojo-rbx/rojo/blob/master/plugin/src/App/StatusPages/Settings/Setting.lua#L21-L32)